Microsoft.Toolkit.Win32/Microsoft.Toolkit.Forms.UI.Controls/InkToolbar/InkToolbarCustomToolButton.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.ComponentModel;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
+
+namespace Microsoft.Toolkit.Forms.UI.Controls
+{
+    /// <summary>
+    /// WinForms-enabled wrapper for <see cref="Windows.UI.Xaml.Controls.InkToolbarCustomToolButton"/>
+    /// </summary>
+    [Designer(typeof(InkToolbarToolButtonDesigner))]
+    public class InkToolbarCustomToolButton : WindowsXamlHostBaseExt
+    {
+        internal Windows.UI.Xaml.Controls.InkToolbarCustomToolButton UwpControl { get; set; }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="InkToolbarCustomToolButton"/> class, a
+        /// WinForms-enabled wrapper for <see cref="Windows.UI.Xaml.Controls.InkToolbarCustomToolButton"/>
+        /// </summary>
+        public InkToolbarCustomToolButton()
+            : this(typeof(Windows.UI.Xaml.Controls.InkToolbarCustomToolButton).FullName)
+        {
+        }
+
+        protected InkToolbarCustomToolButton(string name)
+            : base(name)
+        {
+        }
+
+        protected override void OnHandleCreated(EventArgs e)
+        {
+            base.OnHandleCreated(e);
+            UwpControl = XamlElement as Windows.UI.Xaml.Controls.InkToolbarCustomToolButton;
+        }
+
+        protected override void SetContent()
+        {
```

Comment:
Intentionally blank? Why?

Reply:
Stops the internal child element from being set twice to a parent component (UWPControl)
There may be a better way to resolve this, but without these protected methods, an exception will throw in InkToolbar class when firing RelocateChildToUwpControl